app/grandchallenge/core/static/js/datatables.defaults.mjs

```diff
@@ -0,0 +1,11 @@
+$.extend(true, DataTable.defaults, {
+    scrollX: true,
+});
+
+$(document).on("init.dt", event => {
+    // This is a work-around to get the table to resize properly on extra-large Bootstrap viewport
+    setTimeout($(event.target).DataTable().columns.adjust, 1000);
```

Comment:
There must be a better way. Why does this occur? Is it caused by an interaction with `table-responsive`? Does the event need to happen on init or once the table is ready? Is there another signal this could be attached to?

Reply:
This was a leftover I just grandfathered in. Quite possibly it's not needed anymore and was only applicable for the responsive dt. Not even sure how to reproduce it.

Reply:
It was added in #2160 - there is a video in that. If the behaviour can be replicated now without this line then it can be removed. Unless @amickan can remember exactly why it was added?

Reply:
I think it had to do with inline images (thumbnails), and them loading slowly and the resize event firing before the image was fully loaded. It was a race condition that only occurred sometimes. I also see that I added a max width and max height to the thumbnails right after the linked PR, which might already solve this, unsure.

Reply:
Ah. Ok. I think it is no longer applicable since the responsiveness is gone anyway and all results are now shown with a badge + modal.